.jshintrc

```diff
@@ -1,7 +0,0 @@
-{
-  "undef": true,
-  "unused": true,
-  "node": true,
-  "esversion": 6,
-  "eqeqeq": true
-}
```

Comment:
I was planning to introduce eslint in this PR but figured it will trigger a lot of code polish so left it for the next one, but you're totally right! Should've left it until then 👍 ⏲